.github/workflows/rebase_dependencies.yml

```diff
@@ -0,0 +1,19 @@
+name: Automatic Rebase
```

Comment:
I think this type auto rebase aint gonna be thing. so file not needed

Reply:
Removed.